napari/__init__.py

```diff
@@ -1,16 +1,28 @@
-import warnings
+import os
+from pathlib import Path
+from qtpy import API_NAME
 
-vispy_warning = "VisPy is not yet compatible with matplotlib 2.2+"
+if API_NAME == 'PySide2':
+    # Set plugin path appropriately if using PySide2
+    import PySide2
 
-with warnings.catch_warnings():
-    warnings.filterwarnings(
-        "ignore", category=UserWarning, message=vispy_warning
+    os.environ['QT_PLUGIN_PATH'] = str(
+        Path(PySide2.__file__).parent / 'Qt' / 'plugins'
     )
-    from .viewer import Viewer
+elif API_NAME == 'PyQt5':
+    # Set plugin path appropriately if using PyQt5
+    import PyQt5
 
-from ._view import view
+    os.environ['QT_PLUGIN_PATH'] = str(
+        Path(PyQt5.__file__).parent / 'Qt' / 'plugins'
```

Comment:
this entire clause is unnecessary; we only needed the one above because it was to resolve a bug

Reply:
removed